hedera-node/src/main/java/com/hedera/services/ledger/properties/AccountProperty.java

```diff
@@ -78,10 +78,11 @@
 		public BiConsumer<MerkleAccount, Object> setter() {
 			return (a, v) -> {
 				try {
-					a.setBalance((long) v);
+					a.setBalance((long)v);
```

Comment:
the cast should have the original spacing

Reply:
Done.